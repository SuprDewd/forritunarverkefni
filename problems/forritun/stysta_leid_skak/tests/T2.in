g1
b6
