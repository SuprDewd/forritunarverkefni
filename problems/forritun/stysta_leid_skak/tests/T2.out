4
g1 e2 c3 a4 b6
